pyEpiabm/pyEpiabm/tests/test_place.py

```diff
@@ -0,0 +1,70 @@
+import unittest
+import pyEpiabm as pe
+
+
+class TestPlace(unittest.TestCase):
+    """Test the place class.
+    """
+    @classmethod
+    def setUpClass(cls) -> None:
+        cls.pop = pe.Population()
+        cls.pop.add_cells(1)
+        cls.cell = cls.pop.cells[0]
+        cls.pop.cells[0].add_microcells(1)
+        cls.microcell = cls.cell.microcells[0]
+        cls.pop.cells[0].microcells[0].add_people(1)
+        cls.person = cls.pop.cells[0].microcells[0].persons[0]
+
+    def test_construct(self):
+        """Tests constructor method.
```

Comment:
Strictly, this test should go through each attribute you define in the constructor and ensure that they all exist